swap/src/monero/wallet_rpc.rs

```diff
@@ -226,7 +227,7 @@ impl WalletRpc {
         let name = zip
             .file_names()
             .find(|name| name.contains(PACKED_FILE))
-            .expect("zip archive contains monero wallet rpc")
+            .context("zip archive contains monero wallet rpc")?
```

Comment:
With having a `context` here, this message no longer makes sense. It should say why things went wrong: the zip archive **doesn't** contain the `monero-wallet-rpc` binary.